content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -71,14 +73,16 @@ Changes below are included in InfluxDB Enterprise 1.9.5.
 
 ### Bug fixes
 - Improve heap memory usage when HH queue grows.
-- Avoid rewriting `fields.idx` unnecessarily.
 - Do not close connection twice in `DigestWithOptions`.
 - Do not panic on cleaning up failed iterators.
 - Rename ARM RPMs with `yum`-compatible names.
 - Convert ARM arch names for RPMs during builds via Docker.
 - Do not send non-UTF-8 characters to subscriptions.
 - Error instead of panic for statement rewrite failure.
 - Fix `SHOW SHARDS` showing expiration time for shard groups with no expiration.
+- Change default limit for memory usage from 4GB to unlimited.
+  Memory limits were claimed to be configurable, but are in fact hard-coded.
```

Comment:
`Fix that` ?

Reply:
Hm. I think the fix is implied in the second sentence. But this makes me think: a docs update might be needed as well.

Reply:
Is this relevant? https://github.com/influxdata/docs-v2/blob/344720abbb9bf52cc10b512bd5ea66d6b2d47062/content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md#L261-L267

Reply:
Should this be removed since it's not configurable?

Reply:
Ah I see the confusion.

It is configurable as of 1.9.3. The bug was in 1.9.2 where we added the config parameters but did not hook the config parameters up properly. In 1.9.3 everything is properly configurable.

The relevant settings are in https://github.com/influxdata/docs-v2/blob/344720abbb9bf52cc10b512bd5ea66d6b2d47062/content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md#flux-query-management-settings, especially `total-max-memory-bytes`. In 1.9.2, none of these actually worked correctly. In 1.9.3 everything works like you'd expect.